docs/ja/user-guide/local.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: main
+title: ローカルビルド
+category: User Guide
+menu: menu_ja
+toc:
+- title: sd-localとは？
+  url: "#sd-localとは？"
+- title: sd-localのインストール
+  url: "#sd-localのインストール"
+- title: 簡単な使い方
+  url: "#簡単な使い方"
+- title: configコマンド
+  url: "#configコマンド"
+- title: buildコマンド
+  url: "#buildコマンド"
+---
+
+# sd-localとは？
+sd-localではSCMにコードをアップロードせずに、手元でScrewdriverと同じビルドを再現することができます。sd-localは、Command, Template, Metadataといった機能をScrewdriverと同様に利用することができます。
+
+# sd-localのインストール
+## 実行環境について
+sd-local実行には以下のコマンドが必要になります
+- [docker](https://www.docker.com/)
+  - ビルドの実行環境としてdockerコンテナを利用します
+- [git](https://git-scm.com/)
+  - [src-url]オプションを利用する場合に必要になります
```

Comment:
Is this `[src-url]` intended to a link? Or should we use the format like `--src-url` ?

Reply:
I fixed.